crates/nu-std/std/mod.nu

```diff
@@ -2,6 +2,7 @@
 
 export-env {
     use dirs.nu []
+    use env_exporter.nu []
```

Comment:
I hope we can do something better than this workaround

Reply:
i'm really doubting the need of this file altogether :thinking: 

in `dirs.nu`, we have an `export-env` block and it just works without any additional `use`!
> see running `$env.DIRS_LIST` directly from `nu -n` => this comes from `std dirs`

:point_right: this looks to me like this should just go inside `log.nu` as before, right?